charts/influxdb/templates/backup-cronjob.yaml

```diff
@@ -38,6 +38,13 @@ spec:
               secretName: {{ .Values.backup.gcs.serviceAccountSecret | quote }}
           {{- end }}
           {{- end }}
+          {{- if .Values.backup.s3 }}
+          {{- if .Values.backup.s3.credentialsSecret }}
+          - name: aws-credentials-secret
+            secret:
+              secretName: {{ .Values.backup.s3.credentialsSecret | quote }}
+          {{- end }}
+          {{- end }}
```

Comment:
nit: these two if statements seem like they could be combined with `and` operator